Code/C++/Climbing_Stairs.cpp

```diff
@@ -0,0 +1,80 @@
+/*
+	Problem Description :
+		You are climbing a staircase. It takes n steps to reach the top.
+
+		Each time you can either climb 1 or 2 steps. 
+		In how many distinct ways can you climb to the top?
+	Algorithm :
+		Solving Using Dynamic Programming Approach.
+		Sub-problem Formula of each trail : steps[i] = steps[i - 1] + steps[i - 2]
+		Extra Storage Size : No extra storage Required.
+*/
+
+
+#include <iostream>
+using namespace std;
+
+long long ClimbStairs(int N) 
+{
+	/*
+	 	Base Case : at the first stair (steps[0]) the number of ways to get there is 1
```

Comment:
Add remarks via single line comments in between the code.

Reply:
@raksha009  done